main/src/test/java/org/mobilitydata/gtfsvalidator/validator/NoticeDocumentationTest.java

```diff
@@ -77,6 +77,40 @@ public void testThatAllValidationNoticesAreDocumented() {
         .isEmpty();
   }
 
+  @Test
+  public void testThatAllValidationNoticesAreDocumentedWithFirstLine() {
+    List<Class<?>> noticesWithImproperMultilineDocComment =
+        discoverValidationNoticeClasses()
+            .filter(
+                clazz -> {
+                  NoticeDocComments docComments = NoticeSchemaGenerator.loadComments(clazz);
+                  if (docComments.getDocComment() == null) {
```

Comment:
I think this the intended method to test: 
```suggestion
                  if (docComments.getCombinedDocumentation() == null) {
```

Reply:
Actually, docComments.getShortSummary() is sufficient here.  I changed the two affected lines to reflect this.